festim/mesh/mesh_1d.py

```diff
@@ -72,7 +72,9 @@ def check_borders(self, volume_subdomains):
         ):
             raise ValueError("borders dont match domain borders")
 
-    def define_meshtags(self, surface_subdomains, volume_subdomains):
+    def define_meshtags(self, surface_subdomains, volume_subdomains, interfaces=None):
```

Comment:
Might need some better documentation here, but not completely necessary for this PR